skunkworks_crow/src/main/java/org/odk/share/views/ui/instance/InstancesList.java

```diff
@@ -119,7 +119,7 @@ private void onListItemClick(View view, int position) {
 
     @OnClick(R.id.send_button)
     public void send() {
-        Intent intent = new Intent(this, SendActivity.class);
+        Intent intent = new Intent(this, HpSenderActivity.class);
```

Comment:
Same dialog should be displayed showing which mode to choose (Bluetooth or Hotspot).

Reply:
Fixed.